app/components/container/containerController.js

```diff
@@ -197,6 +197,104 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Con
     });
   };
 
+  $scope.recreate = function() {
+    var config = $scope.container.Config;
+    // HostConfig
+    config.HostConfig = $scope.container.HostConfig;
+    // Name
+    config.name = $scope.container.Name.replace(/^\//g, '');
```

Comment:
In my tests, for a container named "container", the json return "/container" for name